jenkins.sh

```diff
@@ -4,6 +4,12 @@
 : "${JENKINS_HOME:="/var/jenkins_home"}"
 : "${COPY_REFERENCE_FILE_LOG:="${JENKINS_HOME}/copy_reference_file.log"}"
 : "${REF:="/usr/share/jenkins/ref"}"
+
+if ! [ -r "${JENKINS_HOME}" -a -w "${JENKINS_HOME}" ]; then
+    echo INSTALL WARNING: User: "${USER}" missing rw permissions on JENKINS_HOME: "${JENKINS_HOME}";
+    echo "    " Jenkins unable to start. To fix, use chown or setfacl -m u:"${USER}":rwx "${JENKINS_HOME}";
+    exit -1;
```

Comment:
```suggestion
    exit 1;
```

The static analysis with [shellcheck](https://www.shellcheck.net/) fails on this line with the following error:

```text
exit -1;
       ^-- SC2242 (error): Can only exit with status 0-255. Other data should be written to stdout/stderr.
```


=> Exit code must be between 0 and 255 as per https://tldp.org/LDP/abs/html/exitcodes.html